### PR TITLE
Instant doc

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @frewsxcv

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by GuillaumeGomez** [comment]
cc @frewsxcv

**Comment by GuillaumeGomez** [comment]
Removed extra commit (page didn't update...).

File: src/libstd/time/mod.rs [hunk 1]
```diff
@@ -117,14 +122,38 @@ pub struct Instant(time::Instant);
...
 
-/// An error returned from the `duration_since` method on `SystemTime`,
-/// used to learn how far in the opposite direction a system time lies.
+/// An error returned from the `duration_since` and `elapsed` methods on
+/// `SystemTime`, used to learn how far in the opposite direction a system time
+/// lies.
+///
+/// # Examples
+///
+/// ```no_run
```
Comment (frewsxcv) [inline src/libstd/time/mod.rs]: Why `no_run`?

Comment (frewsxcv) [inline src/libstd/time/mod.rs]: Oh, there's a `sleep`, nevermind.

Comment (GuillaumeGomez) [inline src/libstd/time/mod.rs]: :D

**Comment by alexcrichton** [comment]
@bors: retry

* network failure

**Pull request merged**